src/utils/fs.rs

```diff
@@ -110,10 +107,8 @@ pub fn try_infer_extension(path: &Path) -> Option<Extension> {
         Some(Extension::new(&[Gzip], "gz"))
     } else if is_bz2(&buf) {
         Some(Extension::new(&[Bzip], "bz2"))
-    } else if is_xz(&buf) {
-        Some(Extension::new(&[Lzma], "xz"))
-    } else if is_lz(&buf) {
-        Some(Extension::new(&[Lzma], "lz"))
+    } else if is_lzma(&buf) {
+        Some(Extension::new(&[Lzma], "lzma")) // lzma can be `xz` or `lzma`
```

Comment:
`xz` extensions seem to be more popular, should we use `"xz"` instead?

Reply:
That makes sense